packages/video_player/video_player_avfoundation/ios/Classes/messages.g.m

```diff
@@ -0,0 +1,555 @@
+  FLTMixWithOthersMessage *pigeonResult = [[FLTMixWithOthersMessage alloc] init];
+  pigeonResult.mixWithOthers = dict[@"mixWithOthers"];
+  if ((NSNull *)pigeonResult.mixWithOthers == [NSNull null]) {
+    pigeonResult.mixWithOthers = nil;
+  }
+  return pigeonResult;
+}
+- (NSDictionary *)toMap {
+  return [NSDictionary
+      dictionaryWithObjectsAndKeys:(self.mixWithOthers ? self.mixWithOthers : [NSNull null]),
+                                   @"mixWithOthers", nil];
+}
+@end
+
+@interface FLTVideoPlayerApiCodecReader : FlutterStandardReader
+@end
+@implementation FLTVideoPlayerApiCodecReader
+- (nullable id)readValueOfType:(UInt8)type {
+  switch (type) {
+    case 128:
+      return [FLTCreateMessage fromMap:[self readValue]];
+
+    case 129:
+      return [FLTLoopingMessage fromMap:[self readValue]];
+
+    case 130:
+      return [FLTMixWithOthersMessage fromMap:[self readValue]];
+
+    case 131:
+      return [FLTPlaybackSpeedMessage fromMap:[self readValue]];
+
+    case 132:
+      return [FLTPositionMessage fromMap:[self readValue]];
+
+    case 133:
+      return [FLTTextureMessage fromMap:[self readValue]];
+
+    case 134:
+      return [FLTVolumeMessage fromMap:[self readValue]];
+
+    default:
+      return [super readValueOfType:type];
+  }
+}
+@end
+
+@interface FLTVideoPlayerApiCodecWriter : FlutterStandardWriter
+@end
+@implementation FLTVideoPlayerApiCodecWriter
+- (void)writeValue:(id)value {
+  if ([value isKindOfClass:[FLTCreateMessage class]]) {
+    [self writeByte:128];
+    [self writeValue:[value toMap]];
+  } else if ([value isKindOfClass:[FLTLoopingMessage class]]) {
+    [self writeByte:129];
+    [self writeValue:[value toMap]];
+  } else if ([value isKindOfClass:[FLTMixWithOthersMessage class]]) {
+    [self writeByte:130];
+    [self writeValue:[value toMap]];
+  } else if ([value isKindOfClass:[FLTPlaybackSpeedMessage class]]) {
+    [self writeByte:131];
+    [self writeValue:[value toMap]];
+  } else if ([value isKindOfClass:[FLTPositionMessage class]]) {
+    [self writeByte:132];
+    [self writeValue:[value toMap]];
+  } else if ([value isKindOfClass:[FLTTextureMessage class]]) {
+    [self writeByte:133];
+    [self writeValue:[value toMap]];
+  } else if ([value isKindOfClass:[FLTVolumeMessage class]]) {
+    [self writeByte:134];
+    [self writeValue:[value toMap]];
+  } else {
+    [super writeValue:value];
+  }
+}
+@end
+
+@interface FLTVideoPlayerApiCodecReaderWriter : FlutterStandardReaderWriter
+@end
+@implementation FLTVideoPlayerApiCodecReaderWriter
+- (FlutterStandardWriter *)writerWithData:(NSMutableData *)data {
+  return [[FLTVideoPlayerApiCodecWriter alloc] initWithData:data];
+}
+- (FlutterStandardReader *)readerWithData:(NSData *)data {
+  return [[FLTVideoPlayerApiCodecReader alloc] initWithData:data];
+}
+@end
+
+NSObject<FlutterMessageCodec> *FLTVideoPlayerApiGetCodec() {
+  static dispatch_once_t s_pred = 0;
+  static FlutterStandardMessageCodec *s_sharedObject = nil;
```

Comment:
Pigeon fix: These names violate style; it should be `sPred` and `sSharedObject`. Hungarian-style prefixes (`s`, `k`) in ObjC still use normal naming rules.